be/src/exprs/vectorized/runtime_filter_bank.h

```diff
@@ -151,13 +173,29 @@ struct RuntimeBloomFilterEvalContext {
     RuntimeBloomFilterEvalContext() = default;
 
     std::map<double, RuntimeFilterProbeDescriptor*> selectivity;
+    std::vector<RuntimeFilterProbeDescriptor*> descriptors;
     size_t input_chunk_nums = 0;
     int run_filter_nums = 0;
     JoinRuntimeFilter::RunningContext running_context;
     RuntimeProfile::Counter* join_runtime_filter_timer = nullptr;
+    RuntimeProfile::Counter* join_runtime_filter_hash_timer = nullptr;
     RuntimeProfile::Counter* join_runtime_filter_input_counter = nullptr;
     RuntimeProfile::Counter* join_runtime_filter_output_counter = nullptr;
     RuntimeProfile::Counter* join_runtime_filter_eval_counter = nullptr;
+
+    void do_sort_descriptors_if_needed(bool is_sort) {
+        if (selectivity.size() <= 0) {
+            return;
+        }
+
+        descriptors.clear();
+        for (auto& kv : selectivity) {
+            descriptors.push_back(kv.second);
+        }
+        if (is_sort) {
+            std::sort(descriptors.begin(), descriptors.end(), RuntimeFilterProbeDescriptor::Comparator());
```

Comment:
This comparator can not guanrantee rf with the same partition_by_expr_ids be adjacent.